app/main.py

```diff
@@ -14,4 +14,10 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+
+    counter = 0
+    for characters in phrase:
+        if characters.lower() == letter.lower():
+            counter += 1
+
+    return counter
```

Comment:
try to write it in one line

Reply:
i wrote code in one line, but then i try to commit changes, i have massage from git that nothing to commit. i wrote git add . , but it do nothing.  i cant change branch. what do i do uncorrect?

Reply:
![Screenshot 2024-05-04 at 21 50 26](https://github.com/mate-academy/py-count-occurrences/assets/105336389/74060016-3b92-442b-8d09-0890fe794336)